doc/users/next_whats_new/ecdf.rst

```diff
@@ -0,0 +1,12 @@
+``Axes.ecdf``
+~~~~~~~~~~~~~
+A new Axes method, `~.Axes.ecdf`, allows plotting empirical cumulative
+distribution functions without any binning.
+
+.. plot::
```

Comment:
```suggestion
.. plot::
    :include-source: true
```

or we just get the image without the source!

Reply:
done